tests/installation/usb_install.pm

```diff
@@ -29,7 +29,9 @@ sub run {
     assert_script_run("set -o pipefail");
 
     # Specify the two usb drives to use
-    my @usb = split('\n', script_output("ls /dev/disk/by-id/ -l | grep -i usb | grep -i -v -E \"generic|part\" | sed 's#^.*\\\/##'"));
+    my $cmd = "ls /dev/disk/by-id/ -l | grep -i usb | "
+      . "grep -i -v -E \"generic|part|Virtual_NetFile\" | sed 's#^.*\\\/##'";
```

Comment:
Why have to exclude virtual_netfile ?

Reply:
That's empty usb device, just like those with `generic ...`

Reply:
What I observed on vh081/082 is that not only one virtual devices listed, so you have to change the keyword to filter more Virtual devices.
```
Linux_Virtual_NetFile
Linux_Virtual_CDROM
```

Reply:
Updated.